docs/team/ainsleyj.md

```diff
@@ -13,14 +13,69 @@ Given below are my contributions to the project.
 
-* **Contributions to the DG**: To be added.
+      * Converted the fields from AB3 to the fields in Harmonia. For example, the field `Deadline` and `Description` were added.
+      * Changed the parsers and prefixes, to support the `AddCommand`.
+      * Renamed some files and methods, to support the functionalities in Harmonia.
+      * Refactored the tests related to `AddCommand`.
 
-* **Contributions to team-based tasks**: To be added.
+  2. Modified the `DeleteCommand` and `ListCommand` in AB3, such that they work correctly for the task list. Related PRs: [#122](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/122) and [#123](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/123).
 
-* **Review/mentoring contributions**: To be added.
+  3. Refactored the messages in Harmonia. Related PRs: [#124](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/124), [#261](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/261), and [#262](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/262).
 
-* **Contributions beyond the project team**: To be added.
+  4. Enhanced the `find` command, such that it supports:
+
+      * Search by a range of deadline. Related PR: [#177](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/177)
+      * Search by description, by keyword matching. Related PRs: [#198](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/198) and [#214](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/214).
+      * Search by priority. Related PR: [#198](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/198).
+      * Search by completion status. Related PR: [#218](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/218).
+
+* **Contributions to the UG**:
+
+  1. Updated part 3.3 - 3.5 (list all tags, delete and find). Related PR: [#192](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/192) and [#210](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/210).
+
+  2. Added glossary to the UG. Related PR: [#210](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/210).
+
+  3. Added a summary of constraints on user inputs to the UG, and refactor some instructions on commands. Related PR: [#263](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/263).
+
+* **Contributions to the DG**:
+
+  1. Added and formatted the use cases 5 - 9. Related PR: [#65](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/65).
+
+  2. Updated the part related to search by date. Related PR: [#166](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/166).
+
+* **Contributions to team-based tasks**:
+
+  1. Added trial release [v1.3.trail](https://github.com/AY2122S2-CS2103T-T09-1/tp/releases/tag/v1.3.trial). Related PR: [#170](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/170).
```

Comment:
Should this be `trial` instead of `trail`?

```suggestion
  1. Added trial release [v1.3.trial](https://github.com/AY2122S2-CS2103T-T09-1/tp/releases/tag/v1.3.trial). Related PR: [#170](https://github.com/AY2122S2-CS2103T-T09-1/tp/pull/170).
```